memgpt/schemas/block.py

```diff
@@ -62,7 +62,7 @@ class Block(BaseBlock):
     """Block of the LLM context"""
 
     id: str = BaseBlock.generate_id_field()
-    value: str = Field(..., description="Value of the block.")
+    value: Union[str, List[str]] = Field(..., description="Value of the block.")
```

Comment:
I think handling `List[str]` is kind of annoying, since we have to handle those cases differently. and you can represent a List as a string.

Reply:
but might be fine to leave for now

Reply:
OK, so I'll drop support for it?

Reply:
"drop support" = correct the typing hints in `memory.py`